scripts/clusterCreation/deployASB.sh

```diff
@@ -0,0 +1,609 @@
+#!/bin/bash
```

Comment:
Should this file be here? I believe this might be from older automation work that was already merged.

Reply:
No, actually should not, thanks.